backend/platform_settings_v2/platform_auth_helper.py

```diff
@@ -0,0 +1,51 @@
+import logging
+
+from account_v2.authentication_controller import AuthenticationController
+from account_v2.models import Organization, PlatformKey, User
+from platform_settings_v2.exceptions import KeyCountExceeded, UserForbidden
+from tenant_account_v2.models import OrganizationMember
+
+PLATFORM_KEY_COUNT = 2
+
+logger = logging.getLogger(__name__)
+
+
+class PlatformAuthHelper:
+    """Class to hold helper functions for Platform settings authentication."""
+
+    @staticmethod
+    def validate_user_role(user: User) -> None:
+        """This method validates if the logged in user has admin role for
+        performing appropriate actions.
+
+        Args:
+            user (User): Logged in user from context
+        """
+        auth_controller = AuthenticationController()
+        try:
+            member: OrganizationMember = (
+                auth_controller.get_organization_members_by_user(user=user)
+            )
+        except Exception as error:
+            logger.error(
+                f"Error occurred while fetching organization for user : {error}"
+            )
+            raise error
```

Comment:
@muhammad-ali-e do we need the stack and trace here? Or will raising the error again will take care of it?

Reply:
Removed unwanted handling generic exception